### ISSUE TITLE
wreckrun: --input different for each task works

### ISSUE THREAD

**Issue by chu11** [flux-framework/flux-core#1947]
In #1946 hit

```
FAIL: t2000-wreck.t 15 - wreckrun: --input different for each task works
             t2000-wreck.t:  FAIL: N=73  PASS=71  FAIL=1 SKIP=1 XPASS=0 XFAIL=0
```

with the debug output of

```
expecting success: 
        for i in `seq 0 3`; do echo "hi $i" > i.$i; done &&
        flux wreckrun --input="i.0:0;i.1:1;i.2:2;i.3:3" -l -n4 cat |
		sort -n > output.multi &&
        cat >expected.multi <<-EOF &&
	0: hi 0
	1: hi 1
	2: hi 2
	3: hi 3
	EOF
        test_cmp expected.multi output.multi

--- expected.multi	2019-01-22 21:31:31.304176846 +0000
+++ output.multi	2019-01-22 21:31:31.304176846 +0000
@@ -1,4 +1,3 @@
 0: hi 0
 1: hi 1
-2: hi 2
-3: hi 3
+2: hi 3
not ok 15 - wreckrun: --input different for each task works
```

This debug output is fairly disturbing, as it ran only 3 tasks and one task was given the wrong input file.

**Comment by chu11** [comment]
@garlick notes also seen in #1931

**Comment by grondo** [comment]
I was able to reproduce other strange errors with per-task input for wreck jobs. My current theory is that this broke with introduction of the `nokz` style IO, though I have no idea why. Part of me wants to just force `-o kz` on this test and move along. I doubt anyone uses that per-task input format anyway...

**Comment by grondo** [comment]
closed by #1988
